File: scratch/sample_invmst.php
<?php
$python = 'C:\\python32\\python.exe';
$script = <<<'PY'
import pyodbc

conn = pyodbc.connect("DSN=mms_as400;UID=maindss;PWD=windss", autocommit=True)
cur = conn.cursor()
cur.execute("SELECT INUMBR, ICHECK, IDESCR, INLRTL FROM MMLTSLIB.INVMST WHERE INUMBR IN (10000001, 10000002, 10000010, 10000022, 10000023)")
for r in cur.fetchall():
    print(f"INUMBR={r[0]}, ICHECK={r[1]}, IDESCR={r[2].strip()}, INLRTL={r[3]}")
conn.close()
PY;

$pyFile = __DIR__ . '/test_prices.py';
file_put_contents($pyFile, $script);
$cmd = '"' . $python . '" "' . $pyFile . '" 2>&1';
$output = shell_exec($cmd);
echo "<pre>" . htmlspecialchars($output) . "</pre>";

File: scratch/sample_invmst.py
import os
import pyodbc
import json

out_file = os.path.join(os.path.dirname(__file__), "invmst_sample.txt")

with open(out_file, "w", encoding="utf-8") as f:
    try:
        conn = pyodbc.connect("DSN=mms_as400;UID=maindss;PWD=windss", autocommit=True)
        cur = conn.cursor()
        
        cur.execute("SELECT * FROM MMLTSLIB.INVMST FETCH FIRST 5 ROWS ONLY")
        cols = [d[0] for d in cur.description]
        f.write(f"Columns in INVMST: {cols}\n\n")
        
        rows = cur.fetchall()
        for i, r in enumerate(rows, 1):
            r_dict = dict(zip(cols, r))
            f.write(f"--- Sample Row {i} ---\n")
            for k in cols[:30]:
                f.write(f"  {k}: {r_dict.get(k)}\n")
        conn.close()
    except Exception as e:
        f.write(f"Error: {e}\n")